pandas-stubs/core/tools/datetimes.pyi

```diff
@@ -94,7 +91,7 @@ def to_datetime(
     unit: str | None = ...,
     origin: Literal["julian", "unix"] | TimestampConvertibleTypes = ...,
     cache: bool = ...,
-) -> TimestampSeries: ...
+) -> Series[Timestamp]: ...
```

Comment:
I think you may need to flip the order of this class because the declaration of `Series` otherwise `ty` does not find it.